mindmeld/components/question_answerer.py

```diff
@@ -304,18 +367,41 @@ def transform(doc):
                     app_namespace,
                 )
 
+        def _generate_mapping_data(embedder_model, embedding_fields):
+            # generates a dictionary with any metadata needed to create the mapping"
+            MAX_ES_VECTOR_LEN = 2048
```

Comment:
is this an ES hard limit?

Reply:
I believe so: https://www.elastic.co/guide/en/elasticsearch/reference/current/dense-vector.html